### PR TITLE
add deprecation to README.md

### PR DESCRIPTION
**Pull request by pdpinch**
### What are the relevant tickets?

closes https://github.com/mitodl/hq/issues/6233

### Description (What does it do?)

Adds a deprecation warning in advance of archiving the repo

### CODE REVIEW

**Comment by pdpinch** [comment]
@arslanashraf7 is this enough to archive this repo?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+⛔️ DEPRECATION WARNING
+======================
+
+This repository is deprecated. Please see [open-edx-plugins](https://github.com/mitodl/open-edx-plugins) for all the future updates and development.
```
Comment (arslanashraf7) [inline README.md:4]: The username changer is still in its own repo. I'll create a ticket to migrate it to https://github.com/mitodl/open-edx-plugins.

```suggestion
This repository is deprecated. Please see [edx-username-changer](https://github.com/mitodl/edx-username-changer) for all the future updates and development.
```

**Review by arslanashraf7 — APPROVED** [review]
👍

**Comment by arslanashraf7** [comment]
> @arslanashraf7 is this enough to archive this repo?

imo, Yes.

**Pull request merged**